openupgrade_scripts/scripts/account/17.0.1.2/pre-migration.py

```diff
@@ -234,15 +234,17 @@ def _pre_create_account_report_active(env):
     """
     Precreate column with default value true, then switch back to false
     """
-    env.cr.execute(
+    openupgrade.logged_query(
+        env.cr,
         """
         ALTER TABLE account_report ADD COLUMN active boolean DEFAULT true
-        """
+        """,
     )
-    env.cr.execute(
+    openupgrade.logged_query(
+        env.cr,
         """
         ALTER TABLE account_report ALTER COLUMN active SET DEFAULT false
```

Comment:
```suggestion
        ALTER TABLE account_report ALTER COLUMN active DROP DEFAULT
```

or you can use `add_fields` from openupgradelib

Reply:
I want to avoid the part of ir_model_fields and ir_model_data inside `add_fields`

Reply:
Why? The ORM is doing later the same.

Reply:
Exactly. Thus, why bother?

Reply:
For not having 8 lines of code where you can have 1.